uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
Good point @bugobliterator.

@EShamaev do we expect that raw air measurements will be typically produced by the same physical device? If that is not entirely likely, perhaps we should consider separation of this message into two smaller ones, one for the barometric data and the other for the differential pressure data? Although that would deprive us of the covariance matrix.